connector/doc/project/contribute.rst

```diff
@@ -59,3 +59,41 @@ New projects should ideally respect it.
 .. _`OpenERP Magento Connector`: https://github.com/OCA/connector-magento
 .. _`OpenERP Prestashop Connector`: https://github.com/OCA/connector-prestashop
 .. _`E-Commerce Connector`: https://github.com/OCA/connector-ecommerce
+
+*************************************************
+Creating or maintaining a translation of this doc
+*************************************************
+
+- Install Odoo, its dependencies, sphinx, sphinx_bootstrap_theme and
+  sphinx-intl
+- Add `this patch
+  <https://bitbucket.org/shimizukawa/sphinx-intl/pull-request/9/>`_
+  to sphinx-intl (until merged) to support *fuzzy* translations
+- create an empty database with the connector module installed
+- ``cd connector/doc``
+- rebuild the gettext .pot source catalogs: ``make gettext``
+- update the .po translation files from the latest .pot files (here for
+  language 'fr'): ``sphinx-intl update -l fr -p _build/locale``
+- create or edit the translation in the .po files: ``poedit
+  locale/fr/LC_MESSAGES/*.po``
+- compile the .po files into .mo files: ``sphinx-intl build``
+- build the translated documentation to html: ``make SPHINXOPTS="-Dlanguage=fr"
+  html``
+
+The same using a `buildout
+<https://bitbucket.org/anybox/public_buildbot_buildouts/src/tip/odoo-connector.cfg>`_::
+
+    $ mkdir buildout && cd buildout
+    $ wget https://bitbucket.org/anybox/public_buildbot_buildouts/raw/tip/odoo-connector.cfg
+    $ wget https://bitbucket.org/anybox/public_buildbot_buildouts/raw/tip/bootstrap.py
+    $ python bootstrap.py && bin/buildout -c odoo-connector.cfg
```

Comment:
Maybe repeat this step here not to miss it:
`Add `this patch <https://bitbucket.org/shimizukawa/sphinx-intl/pull-request/9/>`_ to sphinx-intl (until merged) to support *fuzzy* translations`